packages/cli-doctor/src/tools/healthchecks/gradle.ts

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import {HealthCheckInterface} from '../../types';
+import {findProjectRoot} from '@react-native-community/cli-tools';
+import {logError} from './common';
+
+const label = 'Gradlew';
+const description = 'Build tool required for Android builds';
+
+const platform = process.platform as 'darwin' | 'win32' | 'linux';
+
+export default {
+  label,
+  description,
+  getDiagnostics: async () => {
+    const root = findProjectRoot();
+    const filename = platform === 'win32' ? 'gradlew.bat' : 'gradlew';
+
+    const gradleWrapperFile = path.join(root, `android/${filename}`);
```

Comment:
we should be able to get `android/` folder from `config.platforms.android.sourceDir`, instead of hardcoding it